packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanelMain.tsx

```diff
@@ -193,7 +193,7 @@ function TokenBalance({
 
 function BalancesContainer({ children }: { children: React.ReactNode }) {
   return (
-    <div className="ml-1 flex flex-col flex-nowrap items-start break-all text-sm font-extralight tracking-[.25px] text-white md:items-end md:text-lg lg:font-medium">
+    <div className="ml-1 flex flex-col flex-nowrap items-start items-end break-all text-sm font-extralight tracking-[.25px] text-white md:text-lg lg:font-medium">
```

Comment:
We can also remove `items-start`

```suggestion
    <div className="ml-1 flex flex-col flex-nowrap items-end break-all text-sm font-extralight tracking-[.25px] text-white md:text-lg lg:font-medium">
```

Reply:
Hey, thanks for the suggestion. 
I've made change.